go/consensus/tendermint/abci/gas.go

```diff
@@ -69,7 +77,7 @@ func NewGasAccountant(maxUsedGas transaction.Gas) GasAccountant {
 
 type nopGasAccountant struct{}
 
-func (ga *nopGasAccountant) UseGas(op transaction.Op, costs transaction.Costs) error {
+func (ga *nopGasAccountant) UseGas(multiplier int, op transaction.Op, costs transaction.Costs) error {
```

Comment:
Should this still check parameter sanity?

Reply:
Good idea, added.